charts/k8ssandra/templates/cassandra/cassdc.yaml

```diff
@@ -166,11 +172,12 @@ spec:
 {{- end }}
   podTemplateSpec:
     spec:
+      securityContext: {{- ternary ($cassPodSecurityCtx) (.Values.cassandra.podSecurityContext | toYaml) (empty .Values.cassandra.podSecurityContext) | nindent 8 }}
```

Comment:
Following from my comment above about just setting defaults vs templates, I think this could then be simplified to 

```yaml
securityContext: {{- .Values.cassandra.podSecurityContext | toYaml | nindent 8 }}
```
And when the changes from https://github.com/k8ssandra/cass-operator/pull/218 are available we won't even need to set the `securityContext`.

Reply:
See comment above for more explanation on the use of templates.  I'm not clear on why securityContext would no longer need to be set.  Are you saying for the pod in this specific context, or for all of the medusa, reaper, init containers, etc. in the cassdc  spec overall?   The entire scope of these changes are securityContext k8s related.

Reply:
Moving forward with changes to remove from templates and apply to directly to the `values.yaml` for the configuration by users.